core/src/edx/org/openedx/core/ui/theme/Colors.kt

```diff
@@ -91,6 +91,8 @@ val light_primary_card_caution_background = Color(0xFFF3F1ED)
 val light_primary_card_info_background = Color(0xFFE7E4DB)
 
 val light_inbox_time_marker_color = Color(0xFF707070)
+val light_notification_primer_badge = Color(0xFF5DE3C0) // Accent 500
+val light_notification_primer_card_background = light_primary_card_info_background
```

Comment:
The color code doesn't match the design.

Reply:
I have updated the Primer badge color, the background color (#E7E4DB) is alright.